development_gdp.env

```diff
@@ -149,13 +140,14 @@ EXTRA_USERPAGE_STYLES=""
 
 # Which svn repo and version of migrid should be used
 MIG_SVN_REPO=https://svn.code.sf.net/p/migrid/code/trunk
-MIG_SVN_REV=5779
+MIG_SVN_REV=5787
```

Comment:
We should really drop svn tag as well if dropping git rev. The former is just the legacy way of picking the `migrid` codebase from the sourceforge repo, which we sync to `migrid-sync` at github. It may still make sense to tag the image with a migrid version or a plain timestamp to make it more clear if it's a current version during debug, bug reports, etc.

Reply:
Sure, I left the SVN examples as comments in development.env and production.env
The development_gdp.env is using SVN to checkout the migrid code, is that on purpose or should we changed that to GIT as well?

Reply:
Alright, the gdp containers should move to git `edge`. 
In any case what I noticed was the following line with 
`CONTAINER_TAG=":${MIG_SVN_REV}"`
which we don't want. Updating accordingly to `MIG_GIT_BRANCH` is fine.